packages/amplication-server/prisma/schema.prisma

```diff
@@ -8,17 +8,17 @@ datasource db {
 }
 
 model Account {
-  id            String   @id @default(cuid())
-  createdAt     DateTime @default(now())
-  updatedAt     DateTime @updatedAt
-  email         String   @unique(map: "Account.email_unique")
-  firstName     String
-  lastName      String
-  password      String
-  currentUserId String?  @unique(map: "Account.currentUserId_unique")
-  githubId      String?
-  currentUser   User?    @relation(fields: [currentUserId], references: [id], onDelete: SetNull, onUpdate: Cascade)
-  users         User[]   @relation("AccountOnUser")
+  id               String            @id @default(cuid())
```

Comment:
we need to make sure we all use the same formating - it looks like there is no changes in this model and still there are new extra spaces